plugins/inputs/execd/execd.go

```diff
@@ -124,7 +124,7 @@ func (e *Execd) cmdReadOutStream(out io.Reader) {
 	for {
 		metric, err := parser.Next()
 		if err != nil {
-			if err == influx.EOF {
+			if err == influx.ErrEOF {
```

Comment:
Can we please use `errors.Is(err, influx.ErrEOF)` to gracefully handle wrapped errors?